packages/sdk/src/contracts/interfaces.contract.ts

```diff
@@ -24,14 +24,14 @@ export enum InterfaceType {
   ERC721, //    2
   ERC1155, //   3
   ROYALTIES, // 4
-  GENERIC, //    5
+  GENERIC, //   5
```

Comment:
Add comment that links to solidity code: https://github.com/holographxyz/holograph-protocol/blob/develop/contracts/enum/InterfaceType.sol